src/main/java/ay2021s1_cs2103_w16_3/finesse/ui/SavingsGoalCard.java

```diff
@@ -0,0 +1,79 @@
+package ay2021s1_cs2103_w16_3.finesse.ui;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import ay2021s1_cs2103_w16_3.finesse.model.budget.MonthlyBudget;
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.StringBinding;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Region;
+
+public class SavingsGoalCard extends UiPart<Region> {
+    private static final String FXML = "SavingsGoalCard.fxml";
+    private final Image savingsPanelPicture = new Image(this.getClass()
+            .getResourceAsStream("/images/SavingsImage.png"));
+
+    @FXML
+    private Label monthlyExpenseLimit;
+    @FXML
+    private Label monthlySavingsGoal;
+    @FXML
+    private Label budgetStatus;
+    @FXML
+    private Label savingsStatus;
+    @FXML
+    private ImageView savingsPicture;
+
+    /**
+     * Constructor of SavingsGoalCard.
```

Comment:
Useless comment.

Reply:
Removing this comment fails checkstyle.

Reply:
Try to make it more descriptive, I don't need a comment to tell me this is a constructor.